### DISCUSSION TITLE
Where is help

### DISCUSSION THREAD

**Discussion by akujbida** [keepassxreboot/keepassxc#9772] [Q&A] [answered]
Giihub is usually dev oriented so I'm guessing this isn't a help forum. Is there one?

**Comment by michaelk83** [comment, marked as answer]
The Discussions section where you posted this is the closest there is to a help forum. There is also a [Wiki](https://github.com/keepassxreboot/keepassxc/wiki), [FAQ](https://keepassxc.org/docs/), and [User Guide](https://keepassxc.org/docs/KeePassXC_UserGuide.html).
You can also search previous issues (don't forget to include the closed ones!), as they often contain solutions to specific users' problems.

**Reply by akujbida** [reply]
Thanks very much. I have searched here already, read the resources you mention. I'll try posting my issues here.
